docs/using/config/configuration.rst

```diff
@@ -302,9 +302,11 @@ Deployment Configuration
 The deployment configuration specifies which applications are run on what nodes.
 It consists of a version and a mapping from node names to application names.
 
-The required parameters are ``version`` and ``applications``.
+The required parameters are ``version`` and ``nodes``.
 For now the ``version`` must be ``1``.
 
+Each hostname in the deployment configuration should be a list of application names that match those defined in the application configuration.
```

Comment:
I think another branch changes the example to stop using a hostname and use an IP address instead (which is the only thing that works).  So it would be nice to say something other than "hostname" here.  "Node address"?

Also the sentence reads oddly.  The hostname isn't a list of application names.  It's a key in a mapping and the associated value is a list of application names.